src/posts/keymaps/argenkiwi.md

```diff
@@ -0,0 +1,24 @@
+---
+author: argenkiwi
+baseLayouts: [QWERTY, Colemak, Dvorak]
```

Comment:
Why are Colemak and Dvorak listed? All I see is QWERTY in both the source files and the images. I realize that the combos and layer keys are all positional so they are not coupled to the base alpha layout and one could swap QWERTY with nearly any other alpha layout and *kenkyo* would still work but if we start listing all possible base layouts, there would be no end to this. What do *you* use personally?

Reply:
I use Colemak and QWERTY personally. In that case I would list it as a QWERTY layout which was the original intended target.

Reply:
Also, I selected English and Spanish as those are the languages I use it with, but if this refers to the actual Spanish layout I'd be happy to go with English only.